server/setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 
 NAME = "openapi_server"
-VERSION = "1.1.1"
+VERSION = "1.0.0"
```

Comment:
@thomasyu888 Shouldn't it be `1.1.2`?

Reply:
I thought so too, but that is the version of the application itself, which technically doesn't have to be the same as the schemas.  Are we going to release a tag per schema release?  I thought about doing this automatically, but if we have any schemas patch updates, it gets ugly.

Reply:
You're right. We don't don't care about the version of the stub, just the version of the schema implemented.